src/main/java/org/opentripplanner/middleware/tripMonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -46,29 +48,49 @@
     private static final Logger LOG = LoggerFactory.getLogger(CheckMonitoredTrip.class);
 
     public final MonitoredTrip trip;
+
     /**
      * Used to track the various check trip notifications and construct email/SMS messages.
      */
     public final Set<TripMonitorNotification> notifications = new HashSet<>();
+
     /**
-     * The index of the matching {@link Itinerary} as found in the OTP {@link OtpResponse} planned as part of this check.
+     * The matching itinerary from the previous run of this job.
      */
-    public Itinerary matchingItinerary;
+    public Itinerary previousMatchingItinerary;
+
     /**
-     * The OTP response planned to check the stored {@link Itinerary} against.
+     * The matching {@link Itinerary} that is used during this check. It is calculated in the following ways:
+     * - the trip's Itinerary if this check is being ran for the first time.
+     * - the trip's JourneyState if the check has already been performed
+     * - an OTP response if the trip is currently active
      */
-    public OtpResponse otpResponse;
+    public Itinerary matchingItinerary;
+
     /**
      * Tracks the time the notification was sent to the user.
      */
     public long notificationTimestampMillis = -1;
 
+    /**
+     * The journey state that was calculated from the prevous run of this job.
```

Comment:
typo: prevous -> previous

Reply:
Fixed in 74d09031e8b0f80d0116218d4cb6cfb4d509d6a6.